doc/minimal-manual.nix

```diff
@@ -0,0 +1,75 @@
+# Minimal manual prototype
+
+with import <nixpkgs> {};
+
+let
+
+  doc-support = import ./doc-support {};
+
+  rootPath = "http://localhost:8000/result";
+
+  anchors = import ./anchors.nix;
+  rewriteAnchor = a: v: ''s|href="(${a})|href="${rootPath}${v.path}\1|g'';
+  rewriteAnchors = lib.mapAttrsToList rewriteAnchor anchors;
+  rewriteHeader = ''s|# ([^{]*)\s\{([^}]*)}$|# <span id="\2">[\1](\2)</span>|g'';
+  rewriteAdmonition = ''s/::: warning/<div class="warning"><h3 class="title">Warning<\/h3><p>/g' | ${gnused}/bin/sed -r 's/:::/<\/p><\/div>/g'';
+
+  joinRewrites = lib.concatStringsSep "; ";
+
+  sedRewritesPre = joinRewrites [
+    rewriteHeader
+    rewriteAdmonition
+  ];
+
+  sedRewritesPost = joinRewrites rewriteAnchors;
+
+  sedCmdPre = "${gnused}/bin/sed -r '${sedRewritesPre}'";
+  cmarkCmd = "${cmark-gfm}/bin/cmark-gfm --unsafe";
```

Comment:
Could you add a comment why unsafe is needed?

Reply:
If we do not use `--unsafe`, all the raw HTML gets output as:
```
<!-- raw HTML omitted -->
```

Currently, I'm using raw HTML for the admonition and the header ids.